### DISCUSSION TITLE
Thanks to everyone who helped with StaxRip v2.1.6.0 Stable!

### DISCUSSION THREAD

**Discussion by Neltulz** [staxrip/staxrip#422] [General]
I just wanted to make it clear how much I appreciate all of the hard work that went into this latest build of StaxRip.

Shoutout to users Dendraspis 44vince44 Alkl58 JJKylee, dali99, BlueSwordM, and of course stax76 (and any others I might have missed) for all of the AV1 related improvements, bug reports, etc etc.  (Trying to avoid @ bombing 20 people in a single discussion).

Lots of feedback regarding AV1 and Aomenc went into making this version great for AV1 encoding. I'm sure that there are tons of other non-av1 related improvements too that went into this, so shoutout to everyone else who helped there.

I really do appreciate all of the hard work that went into this from both the author and the community.  Thanks for all of your support!

**Comment by ghost** [comment]
Thanks @Neltulz  for providing us with necessary information, which is a very difficult task considering the insufficient documentation of aomenc!
